frontend/src/assets/css/typography.scss

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;600;700;800&display=swap");
+
+:root {
+  --font-family: "Plus Jakarta Sans";
+  --font-weight-regular: 400;
+  --font-weight-semibold: 600;
+  --font-weight-bold: 700;
+  --font-weight-extrabold: 800;
+  --line-height-normal: normal;
+}
+
+.h2,
+.h3,
+.h4,
+.h5,
+.h6,
+.main-bold-tall,
+.main-bold,
+.main-regular,
+.main-regular-uppercase,
+.additional {
```

Comment:
rename the `.main-*** `
to the `.text-***`
What is the purpose of the `.additional` selector? 

Also its better to use `xs`, `sm`, `md`, `base`, `lg`

Reply:
I renamed "additional" to "text-small". It is used for the error text in the input.

Reply:
I renamed "additional" to "text-xs". It is used for the error text in the input.
Should I move `font-size` to variables too?